### PR TITLE
Add testing to master

### PR DESCRIPTION
**Pull request by idot107**
This brings in the updates needed to do our testing units

### CODE REVIEW

File: src/test/java/models/testCard.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        assertEquals(0,g.columns.get(0).cards.size());
+    }
+    //test the cardCollection functions
+   // @Test
+    //public void testCardCollection()
+    //{
+     //   Game g = new Game();
+        
+        
+    /}
```
Comment (wwang126) [inline src/test/java/models/testCard.java]: Unfinished comment. Causes compilation error.

File: src/test/java/models/testDeck.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  {
+  //NOT SURE WHAT TO TEST
+  }
+  
+  @Test
+  public void testShuffle()
+  {
+  //check values before then shuffle then compare again
+  Game g = new Game();
+  Card a = g.columns.get(0).cards.toString();
```
Comment (wwang126) [inline src/test/java/models/testDeck.java]: These should be cols not columns.

File: src/test/java/models/testDeck.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+  Card b = g.columns.get(1).cards.toString();
```
Comment (wwang126) [inline src/test/java/models/testDeck.java]: These should be Strings not Cards as you're trying to store strings in them.

File: src/test/java/models/testGame.java [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+        assertEquals("2Clubs",g.columns.get(0).cards.get(0).toString());
+        assertEquals("3Clubs",g.columns.get(1).cards.get(0).toString());
+        assertEquals("4Clubs",g.columns.get(2).cards.get(0).toString());
+        assertEquals("5Clubs",g.columns.get(3).cards.get(0).toString());
+    }
+
+    @Test
+    public void testRemoveFunction(){
+        Game g = new Game();
+        g.customDeal(0,3,6,9);
```
Comment (wwang126) [inline src/test/java/models/testGame.java]: No custom deal function.

**Comment by idot107** [comment]
This is out of date and a new branch with correct stuff will be pulled in instead

**Pull request closed without merging**